subcommands/users/cmd.go

```diff
@@ -2,12 +2,12 @@ package users
 
 import (
 	"github.com/cheynewallace/tabby"
+	"github.com/foundriesio/fioctl/client"
+	"github.com/foundriesio/fioctl/subcommands"
```

Comment:
Normally golang IDEs put your own imports at the bottom like they were before.

Reply:
`make format` does it, not IDE.

Reply:
OMG - now i know why your golang files differ from how @vkhoroz and I do them. 

Volodya - what do you think?

Reply:
I simply use `make format` before every commit. That's easy.

BTW. the go-linter should fail if there are format errors, isn't it?

Reply:
I mean, IDEs differ (e.g. with Vim the only IDE is my brain); make format is uniform.

Reply:
i have a feeling i've set our formatter up wrong. I don't think the linter cares about the import order. anyway - i guess  `make format` rules for now.